website/docs/components/radio/radio-a11y.md

```diff
@@ -11,11 +11,10 @@ tabs: Design('radio'), A11y('radio-a11y'), API('radio-api'), Example('radio-code
 
 Table: Keyboard support
 
-| Key                                  | Function                                                                                                                                                                                                                                              |
-| ------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `Tab`                                | Moves focus to the checked `radio` button in the `radiogroup`. If a `radio` button isn’t checked, focus moves to the first `radio` button in the group.                                                                                              |
-| `Shift + Tab`                        | Moves focus to the previous focusable element.                                                                                                                                                                                                        |
-| `Space`                              | If the `radio` button with focus isn’t checked, changes the state to `checked`. Otherwise, does nothing. _Note: The state where a radio isn’t checked only occurs on page load._                                                                    |
+| Key                                                   | Function                                                                                                                                                                                                                                              |
+| ----------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Tab`                                                 | Moves focus to the checked `radio` button in the `radiogroup`. If a `radio` button isn’t checked, focus moves to the first `radio` button in the group.                                                                                               |
```

Comment:
let's also add what happens when pressing Tab while already focusing a radio button, because this is probably the most slippery aspect of this component

Reply:
I didn't want want to add such an obvious thing for the first place, but I see the point! I guess, it worth to add this info. Thank you!